module.json

```diff
@@ -27,6 +27,9 @@
     "nanostack": {
       "mbed-mesh-api": "~0.0.1"
     },
+    "/mbed-os/net/stacks/nanostack": {
```

Comment:
That would break backwards-compatibility with old targets.  I don't want to break everything, but certainly adding "nanostack" in the similarto field of a target is now deprecated.